ui/app/templates/components/clients/history.hbs

```diff
@@ -67,12 +59,9 @@
         <Toolbar data-test-clients-filter-bar>
           <ToolbarFilters>
             <CalendarWidget
-              @arrayOfMonths={{this.arrayOfMonths}}
-              @endTimeDisplay={{this.endTimeDisplay}}
-              @endTimeFromResponse={{this.endTimeFromResponse}}
-              @handleClientActivityQuery={{this.handleClientActivityQuery}}
-              @handleCurrentBillingPeriod={{this.handleCurrentBillingPeriod}}
-              @startTimeDisplay={{this.startTimeDisplay}}
+              @startTimestamp={{this.startMonthTimestamp}}
```

Comment:
This is cleaned up so beautifully ✨

Reply:
felt so nice to remove all those extra args!